addons/l10n_es_edi_facturae/models/account_move.py

```diff
@@ -220,6 +220,7 @@ def _l10n_es_edi_facturae_inv_lines_to_items(self, conversion_rate=None):
         taxes = []
         taxes_withheld = []
         invoice_ref = self.ref[:20] if self.ref else False
+        unit_price_decimals = self.env['decimal.precision'].precision_get('Product Price') or 2
```

Comment:
```suggestion
        unit_price_decimals = self.env['decimal.precision'].precision_get('Product Price')
```
If I'm not mistaken, `2` is already returned by `precision_get` if it doesn't find `name`.